ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -356,6 +358,10 @@ abstract contract CoreRelayerDelivery is CoreRelayerBase, IWormholeRelayerDelive
     Wei transactionFeeRefundAmount,
     ForwardInstruction[] storage forwardInstructions
   ) private {
+     //despite being external, we only allow ourselves to call this function (via CALL opcode)
+    //  used as a means to retroactively revert the call if the calls to the relay provider here revert
+    if (msg.sender != address(this))
+      revert RequesterNotCoreRelayer();
```

Comment:
Should this be placed here? This is a private function.

Reply:
ah good catch, fixed